views/components/GraphTable/index.jsx

```diff
@@ -1,62 +1,89 @@
 import React from 'react'
-import {
-  TableBody,
-  TableCell,
-  TableHead,
-  TableRow,
-  Table,
-  Paper,
-  Toolbar,
-  Button,
-} from '@mui/material'
-import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown'
+import { Paper, Button } from '@mui/material'
+import { SaveAlt } from '@mui/icons-material'
+import Table from '../../tables/Table'
+import { SORT_DIRECTION } from '../../../constants'
 
-const GraphTable = ({ graph, onGetCsv }) => {
+import './GraphTable.css'
+
+const GraphTable = ({
+  graph,
+  onGetCsv,
+  page,
+  rowsPerPage,
+  handleChangePage,
+  handleChangeRowsPerPage,
+  comparator,
+  stableSort,
+  sortBy,
+  sortDirection,
+  onSort,
+}) => {
   if (!graph) return null
 
   const { tableColumns, tableRows } = graph.graphTable
+  const headers = [
+    {
+      dataProperty: 'site',
+      label: 'Network',
+      sortable: true,
+    },
+    ...tableColumns.map(({ name }) => {
+      return {
+        dataProperty: name,
+        label: name,
+        sortable: false,
+      }
+    }),
+  ]
+  const visibleRows = React.useMemo(
+    () =>
+      stableSort(tableRows, comparator).slice(
```

Comment:
do we need to pass `stableSort` as a prop? I think we could just pass in the visible rows?

Reply:
[ad000c7](https://github.com/AMP-SCZ/dpdash/pull/695/commits/ad000c7d44f7306e75c7dee6f21c14aa327dc931)